deploy/kubeturbo-operator/helm-charts/kubeturbo/values.yaml

```diff
@@ -6,8 +6,8 @@ replicaCount: 1
 
 # Replace the image with desired version
 image:
-  repository: turbonomic/kubeturbo
-  tag: 8.6.1
+  repository: icr.io/cpopen/turbonomic/kubeturbo
+  tag: 8.7.3
   pullPolicy: IfNotPresent
 #  busyboxRepository: busybox
```

Comment:
Again, can we double check the `busybox` image location? Is it moved to `icr.io` as well? What should be the default values?

Reply:
Busybox work is postponed after discussion with David Orsie and Billy.